src/EventStore.Client.Streams/WriteResultExtensions.cs

```diff
@@ -0,0 +1,12 @@
+namespace EventStore.Client {
+	internal static class WriteResultExtensions {
+		public static IWriteResult OptionallyThrowWrongExpectedVersionException(this IWriteResult writeResult,
+			EventStoreClientOperationOptions options) =>
+			(options.ThrowOnAppendFailure, writeResult) switch {
+				(true, WrongExpectedVersionResult wrongExpectedVersionResult)
+					=> throw new WrongExpectedVersionException(wrongExpectedVersionResult.StreamName,
```

Comment:
When I get a wrong expected server error during the batch append it always shows stream revision as `None`.

For example, in the stream `test` the actual version was 999:

```
Unhandled exception. EventStore.Client.WrongExpectedVersionException: Append failed due to WrongExpectedVersion. Stream: test, Expected version: None, Actual version: None
   at EventStore.Client.WriteResultExtensions.OptionallyThrowWrongExpectedVersionException(IWriteResult writeResult, EventStoreClientOperationOptions options)
```

Reply:
This has been fixed